### ISSUE TITLE
Iridium breaks browser back button on YouTube

### ISSUE THREAD

**Issue by BlueberryFoxtrot** [ParticleCore/Iridium#385]
When I visit youtube.com, click and watch a video, and then click the browser back button to return to the YouTube homepage, the homepage does not load and I apparently remain on the same page. At that point, there is a mismatch between the URL displayed in the URL bar and the page actually displayed. Only when I then click the reload button does the browser render the youtube.com homepage again.

I am not sure and not currently in a position to test if this affects all releases of Iridium on all OS/browser versions, but I have been able to reproduce this using a clean profile with just Ubuntu modifications and Greasemonkey add-ons installed (and security.csp.enable set to false in about:config to install the raw github Iridium.user.js) on 56.0 (64-bit) Firefox on Ubuntu 16.04 LTS. 

This worked before, and stopped working recently, probably due to a change YouTube did on their end.

PS: For avoidance of doubt: Disabling Iridium makes the back button work again on YouTube as well.

PPS: Also, if instead of clicking the browser back button, the user clicks the YouTube icon on the YouTube video page to navigate back to https://www.youtube.com, then the YouTube homepage will load as expected.

**Comment by ParticleCore** [comment]
Can't reproduce. Please follow the instructions for opening a new issue as requested: https://github.com/ParticleCore/Iridium/wiki/Report-a-bug

**Comment by ParticleCore** [comment]
Closing for now, if you still have any problems follow the instructions in my previous reply.

**Comment by BlueberryFoxtrot** [comment]
If I want to get back to this, do I just reply here and will that reopen it, or do I have to refile?

I've still got problems, but they're iffy and intermittent, and I've been trying to figure out why that is, and whether those problems are related to another problem where Iridium seems to forget the pre-set quality and revert to auto-switching, plus videos start too early to avoid stuttering (all of these problems started at the same time). 

Figuring out when these problems show up and what they depend on may require longer-term testing.

**Comment by ParticleCore** [comment]
You return here and depending on whether or not the issue can be confirmed it will be re-opened.

**Comment by BlueberryFoxtrot** [comment]
Thank you. I can say that thus far, I have NOT had the broken back button problem with the Firefox Add-on/Extension; I've thus far ONLY had that problem with the Iridium.user.js user script. 

However, I have only installed the add-on recently, and my findings even with the user script were oddly inconsistent when testing, so I'm waiting until I've gathered more data from normal use – I hesitate to submit my inconsistent observations but might do so if I don't figure out anything better (and at least then you'd know exactly what I've seen, contradictions and warts and all).
